### PR TITLE
[CLOUDGA-13804] Users CLI support

### PR DESCRIPTION
**Pull request by harshitha-yb**
https://docs.google.com/document/d/11KyXXA1YTyTLNzHtZZIoQxOpdE7t106n5Anjk7uop0I/edit?usp=sharing

### CODE REVIEW

File: cmd/user/user.go [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+			logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))
+		}
+
+		userCtx := formatter.Context{
+			Output: os.Stdout,
+			Format: formatter.NewUserFormat(viper.GetString("output")),
+		}
+
+		if len(resp.GetData()) < 1 {
+			fmt.Println("No users found")
```
Comment (shinji62) [inline cmd/user/user.go:72]: don t use fmt ... if you need to exist fatal or error and exit is what you should use.

File: internal/client/client.go [hunk 2]
```diff
@@ -775,8 +775,119 @@ func (a *AuthApiClient) ListTasks() ybmclient.ApiListTasksRequest {
 	return a.ApiClient.TaskApi.ListTasks(a.ctx, a.AccountID)
 }
 
-func (a *AuthApiClient) WaitForTaskCompletion(entityId string, entityType ybmclient.EntityTypeEnum, taskType ybmclient.TaskTypeEnum, completionStatus []string, message string) (string, error) {
+func (a *AuthApiClient) ListAccountUsers() ybmclient.ApiListAccountUsersRequest {
+	return a.ApiClient.AccountApi.ListAccountUsers(a.ctx, a.AccountID)
+}
+
+func (a *AuthApiClient) CreateBatchInviteUserSpec(cmd *cobra.Command) (*ybmclient.BatchInviteUserSpec, error) {
```
Comment (shinji62) [inline internal/client/client.go:782]: Just pass email and role name as parameter no need to send the full cmd

File: internal/client/client.go [hunk 3]
```diff
@@ -775,8 +775,119 @@ func (a *AuthApiClient) ListTasks() ybmclient.ApiListTasksRequest {
...
+	}
+
+	return "", fmt.Errorf("Could not get user data for email: %s", email)
+}
+
+func (a *AuthApiClient) GetUserByEmail(email string) (ybmclient.UserData, error) {
+	userResp, resp, err := a.ListAccountUsers().Email(email).Execute()
+	if err != nil {
+		c, _ := httputil.DumpResponse(resp, true)
+		logrus.Debug(string(c))
```
Comment (shinji62) [inline internal/client/client.go:877]: no need to cast c

File: cmd/user/user.go (hunk 1 above)
Comment (shinji62) [reply cmd/user/user.go:72]: logrus.Info should work I guess

**Comment by harshitha-yb** [comment]
Marking as draft till Shreyas' comments are resolved

**Review by cdavid — APPROVED** [review]
Please validate the list commands (and others) once with `-o json`

**Review by shinji62 — APPROVED** [review]

**Pull request merged**